storage/libradb/src/lib.rs

```diff
@@ -109,12 +109,8 @@ pub struct LibraDB {
 }
 
 impl LibraDB {
-    pub fn open<P: AsRef<Path> + Clone>(
-        db_root_path: P,
-        readonly: bool,
-        prune_window: Option<u64>,
-    ) -> Result<Self> {
-        let column_families = vec![
+    fn column_families() -> Vec<ColumnFamilyName> {
```

Comment:
const fn?

Reply:
It's allocating a `Vec` on the heap.. so can't be const..